src/codelabs/implicit-animations.md

```diff
@@ -191,75 +191,84 @@ to set `opacity` to 1:
+    at the same time. The following example illustrates this.
 
-  Notice also that `AnimatedOpacity` animates a single property: `opacity`.
-  Some implicitly animated widgets can animate many properties, as the following
-  example illustrates.
 {{site.alert.end}}
 
 ## Example: Shape-shifting effect
 
-The following example shows how to use the [AnimatedContainer][] widget to
+The following example shows how to use the [`AnimatedContainer`][] widget to
 animate multiple properties (`margin`, `borderRadius`, and `color`) with
 different types (`double` and `Color`).
-**The example begins with no animation code**&mdash;it starts with a
-[Material App][] home screen that contains:
+**The example begins with no animation code**.
+It starts with a [Material App][] home screen that contains:
 
-* A `Container` with `borderRadius`, `margin`, and `color` properties that are
-  different each time you run the example.
-* A **Change** button that does nothing when clicked.
+- A `Container` widget configured with a
+ `borderRadius`, `margin`, and `color`.
+  These properties are setup to be regenerated 
+  each time you run the example.
+- A **Change** button that does nothing when clicked.
 
 ### Shape-shifting (starter code)
 
-Click the **Run** button to run the example:
+To start the example, click **Run**.
 
 {% include docs/implicit-animations/shape-shifting-starter-code.md %}
 
-
 ### Animate color, borderRadius, and margin with AnimatedContainer
 
-  This section contains a list of steps you can use to add an
-  implicit animation to the [shape-shifting starter code][].
-  After the steps, you can also run the
-  [shape-shifting complete][] example with the changes already made.
-
-In the [shape-shifting starter code][],
-each property in the `Container` widget (`color`,
-`borderRadius`, and `margin`)
-is assigned a value by an associated function (`randomColor()`,
-`randomBorderRadius()`, and `randomMargin()` respectively).
-By using an `AnimatedContainer` widget,
-you can refactor this code to do the following:
-
-* Generate new values for `color`, `borderRadius`,
-  and `margin` whenever the user clicks the **Change** button.
-* Animate the transition to the new values for `color`,
+This section contains a list of steps you can use to add an
+implicit animation to the [shape-shifting starter code][].
+After completing each step, you can also run the
+[complete shape-shifting example][] with the changes already made.
+
+The [shape-shifting starter code][] assigns
+each property in the `Container` widget a random value.
+Associated functions generate the relevant values:
+
+- The `randomColor()` function generates a
+  `Color` for the `color` property
+- The `randomBorderRadius()` function generates a
+  `double` for the `borderRadius` property.
+- The `randomMargin()` function generates a
+  `double` for the `margin` property.
+
+This section uses the `AnimatedContainer` widget to:
```

Comment:
At this point, I'm lost on what "this section" is referring. Perhaps "The following steps"?

```suggestion
The following steps use the `AnimatedContainer` widget to:
```